src/modules/gameHandler.js

```diff
@@ -0,0 +1,35 @@
+async function newGame() {
+  const res = await fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      name: 'MyGame',
+    }),
+  });
+  const data = await res.json();
+  localStorage.setItem('GameID', data.result.split(' ')[3]);
+}
```

Comment:
- [ ] Please use arrow functions instead of the `function` keyword. This is a project requirement 👍